From 0b44b1aa3d66f974e3d255c6e027947f1375b685 Mon Sep 17 00:00:00 2001
From: Stephen Kelly <steveire@gmail.com>
Date: Mon, 25 Apr 2011 22:09:08 +0200
Subject: [PATCH 2/6] Only start akonadi asynchronously on KDE 4.6.2.

Revises cfa404b7188e4c26bddbc9579728f6d25f8cd214 to hopefully fix
the bug seen on fedora.

Please re-test this on 4.6.2 on fedora too. Unfortunately that's the
best I can do because I can't reproduce the issue.

BUG: 268120
---
 kaddressbook/main.cpp  |    2 +-
 kmail/kmmainwidget.cpp |    2 +-
 kmail/main.cpp         |    2 +-
 kontact/src/main.cpp   |    2 +-
 4 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/kaddressbook/main.cpp b/kaddressbook/main.cpp
index 200f8ec..e4cb3f7 100644
--- a/kaddressbook/main.cpp
+++ b/kaddressbook/main.cpp
@@ -49,7 +49,7 @@ int main( int argc, char **argv )
   MainWindow *window = new MainWindow;
   window->show();
 
-#if !KDE_IS_VERSION(4,6,0)
+#if !KDE_IS_VERSION(4,6,2)
   if ( !Akonadi::Control::start( window ) ) {
     //TODO: add message box after string freeze
     kWarning() << "Unable to start Akonadi server, exit application";
-- 
1.7.9.2

